internal/pkg/v2/infrastructure/redis/device_profile.go

```diff
@@ -18,13 +18,48 @@ import (
 
 const DeviceProfileCollection = "v2:deviceProfile"
 
+// deviceProfileStoreKey return the device profile's store key which combines the collection name and object ID
+func deviceProfileStoreKey(id string) string {
+	return fmt.Sprintf("%s:%s", DeviceProfileCollection, id)
+}
+
+// deviceProfileIsExistByName checks whether the device profile is exist by name
```

Comment:
whether the device profile exists by name

Reply:
Modified.